proto/src/determined/api/v1/api.proto

```diff
@@ -21,33 +22,59 @@ import "determined/api/v1/shell.proto";
-        version: "0.1",
-        contact: {
-            name: "Determined AI",
-            url: "https://determined.ai/",
-            email: "community@determined.ai",
-        }
-        license: {
-            name: "Apache 2.0",
-            url: "http://www.apache.org/licenses/LICENSE-2.0.html"
-        }
-    }
-    schemes: [HTTP, HTTPS],
-    external_docs: {
-        description: "Determined AI Documentation",
-        url: "/docs",
-    }
+  info : {
+    title : "Determined API (Beta)",
+    description : "Determined helps deep learning teams train models more "
+                  "quickly, easily share GPU resources, and effectively "
+                  "collaborate. Determined allows deep learning engineers to "
+                  "focus on building and training models at scale, without "
+                  "needing to worry about DevOps or writing custom code for "
+                  "common tasks like fault tolerance or experiment "
+                  "tracking.\n\nYou can think of Determined as a platform that "
+                  "bridges the gap between tools like TensorFlow and PyTorch "
+                  "--- which work great for a single researcher with a single "
+                  "GPU --- to the challenges that arise when doing deep "
+                  "learning at scale, as teams, clusters, and data sets all "
+                  "increase in size.",
+    version : "0.1",
+    contact : {
+      name : "Determined AI",
+      url : "https://determined.ai/",
+      email : "community@determined.ai",
+    }
+    license : {
+      name : "Apache 2.0",
+      url : "http://www.apache.org/licenses/LICENSE-2.0.html"
+    }
+  }
+  schemes : [ HTTP, HTTPS ],
+  external_docs : {
+    description : "Determined AI Documentation",
+    url : "/docs",
+  },
+  security_definitions : {
+    security : {
+      key : "BearerToken",
+      value : {
+        type : TYPE_API_KEY,
+        description : "Bearer token authentication strategy",
+        name : "Authorization",
+        in : IN_HEADER
+      }
+    }
+  },
+  security : {
+    security_requirement : {key : "BearerToken"; value : {}}
+  }
 };
 
 // Determined is the official v1 of the Determined API.
 service Determined {
     // Login the user.
     rpc Login(LoginRequest) returns (LoginResponse) {
         option (google.api.http) = {post: "/api/v1/auth/login" body: "*"};
-        option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {tags: "Authentication"};
+        option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {tags: "Authentication",
+            security: {};};
```

Comment:
**question:** the semi-colon `;` after `security: {}` is probably a typo/unnecessary?

Reply:
either that or the `,` after `tags` needs to be a semi-colon?

Reply:
it's pretty loose they all should probably be semicolons as these are not really JSON blobs and grpc_gateway examples use semicolon. I have a ticket for adding a formatter that should autoformat all of these.

Reply:
I'll go ahead and make sure the parts I change stay consistent with semicolon vs `comma`

Reply:
and I thought JSON was bad... :D